tensorboard/compat/tensorflow_stub/resource_loader.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2015 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Resource management library."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os as _os
+import sys as _sys
+
+from . import tf_inspect as _inspect
```

Comment:
We can just replace tf_inspect with the standard library module here, since we won't be needing to handle any tf_decorators once we also remove those.  So just `import inspect as _inspect`.